ecpy_hqc_legacy/tasks/tasks/instr/rf_tasks.py

```diff
@@ -153,3 +153,28 @@ def i_perform(self, switch=None):
         else:
             self.driver.output = 'Off'
             self.write_in_database('output', 0)
+
+
+class SetPulseModulationTask(InterfaceableTaskMixin, InstrumentTask):
+    """Set the pulse modulation state of the RF source.
+
+    """
+    # pulse modulation state
+    pm_state = Bool(False).tag(pref=True)
```

Comment:
Use a Unicode to allow to evaluate the value (cf SetRFOnOff) with the proper validation